chainregistry.go

```diff
@@ -751,3 +760,43 @@ func initNeutrinoBackend(chainDir string) (*neutrino.ChainService, func(), error
 
 	return neutrinoCS, cleanUp, nil
 }
+
+func parseHeaderStateAssertion(state map[string]string) (
+	*headerfs.FilterHeader, error) {
+
+	if len(state) == 0 || state == nil {
```

Comment:
Nit: both of these checks are essentially the same, a `nil` map has a `0` length.